DSCResources/MSFT_xWebsite/MSFT_xWebsite.psm1

```diff
@@ -439,6 +547,36 @@ function Test-TargetResource
             }
         }
 
+        #Check AuthenticationInfo
+        if ($PSBoundParameters.ContainsKey('SSlFlags') -and (Test-AuthenticationInfo -Site $Website -Name $Name -AuthenticationInfo $AuthenticationInfo)) 
```

Comment:
The `Test-TargetResource` function does not have the `SSlFlags` parameter.
It's completely fine if you intend to add the `SSLFlags` property in future, but in scope of this specific PR it isn't necessary.